Berksfile

```diff
@@ -4,24 +4,24 @@ source 'https://supermarket.chef.io'
 metadata
 
 cookbook 'conda', github: "logicalclocks/conda-chef", branch: "master"
-cookbook 'kagent', github: "logicalclocks/kagent-chef", branch: "master"
+cookbook 'kagent', github: "logicalclocks/kagent-chef", branch: "opensearch"
```

Comment:
can you please reset this to master branch

Reply:
Yep.